.claude/commands/tm/fix-dependencies/fix-dependencies.md

```
@@ -0,0 +1,81 @@
+Automatically fix dependency issues found during validation.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)